src/test/java/giis/eshopcontainers/e2e/functional/common/BaseLoggedClass.java

```diff
@@ -92,22 +92,28 @@ protected static void setupBrowser() {
      * Navigates to the main menu to enter a user session.
      */
     protected void login() throws ElementNotFoundException {
-        Navigation.toMainMenu(driver, waiter);
-        isLogged = true;
-        log.debug("Logging in user {} ", userName);
-        waiter.waitUntil(ExpectedConditions.elementToBeClickable(By.xpath("//a[contains(text(),'Login')]")), "The button searched by xPath //a[contains(text(),'Login')] is not clickable");
-        Click.element(driver, waiter, driver.findElement(By.xpath("//a[contains(text(),'Login')]")));
-
-        waiter.waitUntil(ExpectedConditions.presenceOfElementLocated(By.id("Username")), "The Username login field is not present");
-        WebElement userNameField = driver.findElement(By.id("Username"));
-        waiter.waitUntil(ExpectedConditions.presenceOfElementLocated(By.id("Password")), "The Password login is not present");
-        WebElement userPassField = driver.findElement(By.id("Password"));
-        userNameField.sendKeys(userName);
-        userPassField.sendKeys(password);
-
-        waiter.waitUntil(ExpectedConditions.elementToBeClickable(By.xpath("//button[contains(.,'Login')]")), "The button searched by xpath //button[contains(.,'Login')] is not clickable");
-        Click.element(driver, waiter, driver.findElement(By.xpath("//button[contains(.,'Login')]")));
-        log.debug("Logging in successful for user {}", userName);
+        try {
```

Comment:
Do not catch exceptions in tests, let junit catch exceptions for you. Revert this change.